### DISCUSSION TITLE
speed, handbrake log display disappear

### DISCUSSION THREAD

**Discussion by dongya-bit** [microsoft/AirSim#3220] [Support Q&A] [answered]
After AirSim is packaged successfully;Why did the speed, handbrake log display disappear

**Comment by jonyMarino** [comment, marked as answer]
Hi @dongya-bit! And welcome to Airsim! Please provide more context. I think you are talking about the car. And I've tested with the City release, and I see the log messages. If you pack that commit, you still have log messages or no?

**Reply by ahmed-elsaharti** [reply]
I also did some testing by packaging the Blocks environment and the logs were visible with the `"Car"` SimMode. They can be enabled/disabled through `"LogMessagesVisible": true` in [settings.json](https://microsoft.github.io/AirSim/settings/)
![image](https://user-images.githubusercontent.com/49626509/102411416-89e54080-3fb7-11eb-8644-1dad9bb63dbd.png)

**Reply by jonyMarino** [reply]
Thanks, @ahmed-elsaharti , for your test! You are always very attentive!

**Reply by ahmed-elsaharti** [reply]
@jonyMarino No problem 👍  And I'm finally done with finals so I'll be practically living in the repo (especially since there's a discussion board now) 😁

**Reply by dongya-bit** [reply]
Thank you. I saw the log message

**Comment by Ahmed** [comment]
This a really cool project. I discovered cool projects on GitHub by random
people tagging me. 😁

**Reply by jonyMarino** [reply]
Thanks, @Ahmed! And sorry for bothering you. Feel free to take a look at whatever you want.
